### PR TITLE
Created va_profile_information_v3_service feature flipper for V3::Pro…

### PR DESCRIPTION
**Pull request by RachalCassity**
…fileInformation

**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary

Created va_profile_information_v3_service feature flipper for the new VAProfile V3 Service. 

## Related issue(s)
- department-of-veterans-affairs/va.gov-team#83186

### CODE REVIEW

File: config/features.yml [hunk 1]
```diff
@@ -1256,6 +1256,10 @@ features:
   va_burial_v2:
     actor_type: user
     description: Allows us to toggle between 21-P530 and 21-P530V2
+  va_profile_information_v3_service:
+    actor_type: user
+    description: When enabled, the VAProfile::V3::Service will be enabled for VaProfile::ProfileInformation
+    enable_in_development: false
```
Comment (rmtolmach) [inline config/features.yml:1262]: Why `false`?

Comment (RachalCassity) [reply config/features.yml:1262]: This is a draft and not ready for review

Comment (RachalCassity) [reply config/features.yml:1262]: @rmtolmach This cannot be enabled until the service configuration is complete

**Review by rmtolmach — APPROVED** [review]

**Comment by rmtolmach** [comment]
@RachalCassity the failing tests were from `master`. If you update the branch, they should pass.

**Pull request merged**